packages/react/src/icon/Icon.tsx

```diff
@@ -0,0 +1,39 @@
+/**Denne komponenten displayer ikoner fra material symbols rounded
+ * https://fonts.google.com/icons?icon.set=Material+Symbols&icon.style=Rounded*/
+
+export const Icon = ({
+  icon,
+  size = 24,
+  color = "black",
+  weight = 600,
+  grade = 0,
+  isFilled = false,
+}: {
+  /**The icon from Material symbols you want to display*/
+  icon: string;
+  /**The font size of the icon */
+  size?: 20 | 24 | 40 | 48;
+  /**The color of the icon*/
+  color?: string;
```

Comment:
Jeg tror det kunne vært hensiktsmessig å oppgi farge som et colorScheme og et tall, slik at det blir lettere for brukeren å velge riktig farge.

Reply:
Mener du da f.eks. green[400] osv? Fordi det er mulig, bare vet ikke hvordan jeg kan vise det i storybook. Jeg kan skrive og lage et lite kodeeksempel på det da,

Reply:
Endte opp med dette 
![image](https://github.com/kartverket/kvib/assets/122277136/69b1556e-eb74-4cd6-a2f3-913373eb9d8e)

Reply:
Usikker på denne, tror personlig jeg foretrekker at det bare er en css string slik at den oppfører seg som alle andre ting som tar inn farger. Det er farlig å anta at brukeren _aldri_ skal få nytte for å bruke noe designsystemet ikke har tatt høyde for

Reply:
Det er mulig å ta inn farger på de vanlige måtene i tillegg til denne. Jeg skal oppdatere dokumentasjonen for å få det tydeligere fram 😊

Reply:
Så vidt jeg har forstått er dette (som vist i bildet) den vanlige måten å ta inn farger på i chakra. Jeg synes det fungerer flott, fordi brukeren ikke trenger å huske/copy-paste hex-verdier, men bare kan forholde seg til de fargene som designerne har jobbet frem. Når det er sagt gjør det ingenting at det er mulig å sende inn hex-koder i tillegg, som Anders sier kan det jo komme nye behov som vi ikke har tatt høyde for.